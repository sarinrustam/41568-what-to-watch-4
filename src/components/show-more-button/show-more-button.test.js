import * as React from "react";
import renderer from "react-test-renderer";

import ShowMoreButton from "./show-more-button";

describe(`Render ShowMoreButton`, () => {
  it(`Should ShowMoreButton render correctly`, () => {
    const handlerButtonClick = jest.fn();
    const tree = renderer.create(
        <ShowMoreButton
          onButtonClick={handlerButtonClick}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
