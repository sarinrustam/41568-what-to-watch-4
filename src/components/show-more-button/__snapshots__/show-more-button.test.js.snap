// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowMoreButton Should ShowMoreButton render correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[MockFunction]}
    type="button"
  >
    Show more
  </button>
</div>
`;
